src/engine/basics/GameStates.hxx

```diff
@@ -6,18 +6,17 @@
 #include "../../util/Singleton.hxx"
 #include "../common/enums.hxx"
 
-
-
 class GameStates : public Singleton<GameStates>
 {
 public:
   friend Singleton<GameStates>;
 
-    /**
+  /**
    * @brief Draw UI. This is a temporary variable until the new UI has been implemented.
    * TODO Remove this function afterwards
    */
   bool drawUI = true;
+  bool rectangularRoads = false;
```

Comment:
we should do another refactoring for fields of classes and change them to be m_<fieldName>

Reply:
good catch. added it.